### PR TITLE
Make player tags show Terraforming Control Station. Add a test.

### PR DESCRIPTION
**Pull request by kberg**
Discover that Venus cards have fractional VPs and adjust.

### CODE REVIEW

File: src/client/components/overview/PlayerTags.vue [hunk 1]
```diff
@@ -187,7 +182,15 @@ export default Vue.extend({
     },
     getVPs(tag: InterfaceTagsType) {
       const modifier = this.modifiers[tag];
-      return `${modifier.points}${modifier.fraction ?? ''}`;
+      const integer = Math.floor(modifier.points);
+      const fraction = modifier.points - integer;
+      let vulgarFraction = '';
+      if (fraction === 0.5) {
+        vulgarFraction = '½';
+      } else if (Math.abs(fraction - (1/3)) < 0.001) {
```
Comment (kberg) [inline src/client/components/overview/PlayerTags.vue:190]: This is just the reality of matching to floating points.

File: src/models/ServerModel.ts [hunk 2]
```diff
@@ -204,13 +204,14 @@ export class Server {
 
   public static getCorporationCard(player: Player): CardModel | undefined {
     if (player.corporationCard === undefined) return undefined;
+    const card = player.corporationCard;
     return {
-      name: player.corporationCard.name,
-      resources: player.getResourcesOnCard(player.corporationCard),
+      name: card.name,
```
Comment (kberg) [inline src/models/ServerModel.ts:209]: Most of this local change is actually due to using a local parameter for `player.corporationCard`

**Comment by kberg** [comment]
Fixes #3820

**Comment by kberg** [comment]
image

File: src/client/components/overview/PlayerTags.vue (hunk 1 above)
Comment (bafolts) [reply src/client/components/overview/PlayerTags.vue:190]: Is this going to eventually need to display 1/4, 1/5, 1/6... 1/10? Using a for loop would avoid having to write this out for each fraction. Using `Number.EPSILON` makes sense to me versus using `0.001`. At that point it is self explanatory what is being checked.

Comment (kberg) [reply src/client/components/overview/PlayerTags.vue:190]: Probably not. And if it does, we shouldn't rely on estimating fractional amounts.

**Review by bafolts — APPROVED** [review]

**Pull request merged**